README.md

```diff
@@ -80,6 +80,7 @@ Once you have an element ID, you can run these commands as well:
 |`sendKeys`|string|Set the given value to the element|
 |`getAttribute`|string, `elementId`|Return the value of attribute key `string` of the element represented by `elementId`. If attribute key is not present for the element, method will return null|
 |`getText`|`elementId`|Return the value of the `text` attribute for an element. If the `text` attribute is not present, this method will return null.|
+|`elementEnabled`|`elementId`|Moves the focus on a element having locator xpath as `elementId`. If uanble to focus on the element, the driver will respond with a error.|
```

Comment:
`elementSelected` is usually used to get if the element is selected. So the client uses `element.enabled` for example, thus it is not good to use as an action.
https://github.com/appium/appium/blob/master/packages/base-driver/lib/protocol/routes.js

Instead, I wondered if a new roku extension command such as `driver.execute_script 'roku: selectElement', {elementId: "element id"}` would be more appropriate. https://github.com/headspinio/appium-roku-driver?tab=readme-ov-file#roku-commands

I'd like to get @jlipps 's opinion as well.

Reply:
I was also thinking to change method from 'elementEnabled' to 'elementSelected' as it relates more to focus functionality and we can use isSelected() method.

Reply:
@jlipps kindly review, I think we go with "elementSelected" option for now.